config/crd/bases/reaper.k8ssandra.io_reapers.yaml

```diff
@@ -1176,6 +1176,27 @@ spec:
               initContainerResources:
                 description: Init Container resources.
                 properties:
+                  claims:
+                    description: "Claims lists the names of resources, defined in
+                      spec.resourceClaims, that are used by this container. \n This
+                      is an alpha field and requires enabling the DynamicResourceAllocation
+                      feature gate. \n This field is immutable. It can only be set
+                      for containers."
+                    items:
+                      description: ResourceClaim references one entry in PodSpec.ResourceClaims.
+                      properties:
+                        name:
+                          description: Name must match the name of one entry in pod.spec.resourceClaims
+                            of the Pod where this field is used. It makes that resource
+                            available inside a container.
+                          type: string
+                      required:
+                      - name
+                      type: object
+                    type: array
+                    x-kubernetes-list-map-keys:
+                    - name
+                    x-kubernetes-list-type: map
```

Comment:
question: Are these changes related? Or are these due to an upgraded version?

Reply:
I think it's related to the kubernetes dependency upgrades (from 0.25 to 0.26) that were required by the cass-operator upgrade.

Reply:
yup, that's here: https://kubernetes.io/blog/2022/12/15/dynamic-resource-allocation/